### PR TITLE
[Paper Claim] Rewards modal with placeholder

### PR DESCRIPTION
**Pull request by yattias**
Rewards modal with placeholder to allow testing flow end-to-end

image

image

### CODE REVIEW

File: components/ResearchCoin/ClaimRewardsModal.tsx [hunk 1]
```diff
@@ -0,0 +1,187 @@
...
+  const handleSubmitClaim = () => {
+
+    if (!currentUser) {
+      return;
+    }
+
+    // @ts-ignore Temporarily ignoring due to hard-coding 
+    submitRewardsClaim({
+      paperId,
+      authorshipId: 1762,
```
Comment (yattias) [inline components/ResearchCoin/ClaimRewardsModal.tsx:71]: @koutst for testing you will need to include an id for any record from `paper_authorship` table

**Review by koutst — APPROVED** [review]
Lgtm

File: components/ResearchCoin/ClaimRewardsModal.tsx (hunk 1 above)
Comment (gzurowski) [reply components/ResearchCoin/ClaimRewardsModal.tsx:71]: Nit: Maybe add a `// FIXME` here to make it easier to revisit this?

**Review by gzurowski — APPROVED** [review]
LGTM 👍

**Pull request merged**